zerver/lib/notifications.py

```diff
@@ -452,6 +453,20 @@ def log_digest_event(msg):
     logging.basicConfig(filename=settings.DIGEST_LOG_PATH, level=logging.INFO)
     logging.info(msg)
 
+def followup_email_delay(user):
+    #type: (UserProfile) -> None
+    days_to_delay = 2
+    current_day = datetime.now(pytz.timezone(user.timezone)).isoweekday()
+    if current_day == 5:
+        # If the day of signup is Friday then delay should be till Monday
+        days_to_delay = 3
+    elif current_day == 4:
+        # If the day of signup is Thursday then delay should be till Friday
+        days_to_delay = 1
+
+    # The delay should be 1 hour before the above calculated delay
```

Comment:
Let's add:
Our goal is to maximize the chance that this is near the top of the user's inbox when the user sits down to deal with their inbox, or comes in while they are dealing with their inbox.